engine/consensus/matching/engine.go

```diff
@@ -985,15 +1056,15 @@ func (e *Engine) requestPendingReceipts() error {
 
 	// request missing execution results, if sealed height is low enough
 	if len(missingBlocksOrderedByHeight) > 0 {
-		log.Info().
+		log.Debug().
```

Comment:
>This is redundant with...

I adjusted this into Debug level log (as well as the approvals) since the  "Checking Sealing log" has included these data.